doc/source/dashboard/ray-dashboard.rst

```diff
@@ -0,0 +1,18 @@
+Ray Dashboard
```

Comment:
Why not just have a single documentation page on the dashboard that combines all of the information from the above three pages. Do you think it would be too long?

Reply:
+1, I think it is better to keep it in one page...

Reply:
Three reasons.
1. I personally prefer short pages when I read documentation.
2. I expected each pages will be much longer, but it turns out it was much shorter. 
3. I thought dashboard document could be eventually very long as we will provide lots of different components.

But given the current length, it totally makes sense to put everything within one page. I will merge them.